src/tagmanager/tm_parser.c

```diff
@@ -69,7 +69,7 @@ static GHashTable *subparser_map = NULL;
 	{'u', tm_tag_union_t},       /* union */      \
 	{'v', tm_tag_variable_t},    /* variable */   \
 	{'x', tm_tag_externvar_t},   /* externvar */  \
-	{'h', tm_tag_undef_t},       /* header */     \
+	{'h', tm_tag_header_t},      /* header */     \
```

Comment:
It's called `header` in ctags but maybe we should call it `tm_tag_include_t` to make it clear this is something that's generated for every `#include` directive and not for every header file in project.

Reply:
sounds good